.github/workflows/requirements-test.yaml

```
@@ -0,0 +1,32 @@
+name: Requirements generated tests
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - 'test/**'
+      - 'backend/**'
+      - 'sdk/**'
```

We should run this workflow only when any of the requirements file changes.
But for now, can you please change it to:

```suggestion
      - '.github/workflows/requirements-test.yaml'
```

Otherwise the workflow won't run in this PR.

Once the PR is ready to get merged, we must change it to use the following files:
```
./test/sample-test/requirements.in
./test/kfp-functional-test/requirements.in
./backend/metadata_writer/requirements.in
./backend/requirements.in
./backend/src/apiserver/visualization/requirements.in
./sdk/python/requirements.in
```

---

ok - thanks for your input.